cli/command/network/cmd.go

```diff
@@ -9,7 +9,7 @@ import (
 
 // NewNetworkCommand returns a cobra command for `network` subcommands
 // nolint: interfacer
-func NewNetworkCommand(dockerCli *command.DockerCli) *cobra.Command {
+func NewNetworkCommand(dockerCli command.Cli) *cobra.Command {
```

Comment:
Please also remove the `// nolint: interfacer` line here (and anywhere else you change to use `command.Cli`).

We don't need the `nolint` once we change to use the interface.